### DISCUSSION TITLE
Is this possible in video production?

### DISCUSSION THREAD

**Discussion by zeonjiho** [lllyasviel/IC-Light#118] [Q&A]
Do you think this model can also be used in videos?

**Comment by TanvirHafiz** [comment]
try this https://civitai.com/articles/6103/ic-light-changer-for-videos

**Comment by QuantumStriker** [comment]
It looks like the model discussed could be applied in video production to enhance various stages of content creation. Given the rapid advancements in the industry, companies like [KPI Creatives](https://www.google.com/maps?cid=10637619299654140657) may explore similar technologies to improve video quality, streamline editing, and create immersive experiences. Integrating AI-based models could lead to more efficient production workflows and allow for creative flexibility.
